GeosCore/hco_utilities_gc_mod.F90

```diff
@@ -1706,39 +1712,35 @@ SUBROUTINE Get_GC_Restart( Input_Opt, State_Chm, State_Grid, &
          SpcInfo%Is_InRestart = .FALSE.
       ENDIF
 
-      ! If data is in file, read in as [mol/mol] and convert to
-      ! [kg/kg dry]. Otherwise, set to background value [mol/mol]
-      ! either stored in species database (advected species all levels and
-      ! non-advected species levels in the chemistry grid) or a small number
-      ! (non-advected species levels above the chemistry grid) converted to
-      ! [kg/kg dry]
+      ! If data is in file, read in as [mol/mol]. Otherwise, set to
+      ! background value [mol/mol] either stored in species database
+      ! (advected species all levels and non-advected species levels
+      ! in the chemistry grid) or a small number (non-advected species
+      ! levels above the chemistry grid).
       IF ( SpcInfo%Is_InRestart ) THEN
 
-         ! Print the min & max of each species as it is read from
+         ! Print the min, max, and sum of each species as it is read from
          ! the restart file in mol/mol
          IF ( Input_Opt%amIRoot ) THEN
-            WRITE( 6, 120 ) N, TRIM( SpcInfo%Name ), &
+            WRITE( 6, 530 ) N, TRIM( SpcInfo%Name ), &
                             MINVAL( Ptr3D ), MAXVAL( Ptr3D ), SUM ( Ptr3D(:,:,1:State_Grid%NZ) )
-120         FORMAT( 'Species ', i3, ', ', a8, ': Min = ', es15.9, &
-                    '  Max = ',es15.9, '  Sum = ',es15.9)
          ENDIF
 
-         ! Convert file value [mol/mol] to [kg/kg dry] for storage
          !$OMP PARALLEL DO       &
          !$OMP DEFAULT( SHARED ) &
          !$OMP PRIVATE( I, J, L )
          DO L = 1, State_Grid%NZ
          DO J = 1, State_Grid%NY
          DO I = 1, State_Grid%NX
-            Spc(N)%Conc(I,J,L) = Ptr3D(I,J,L) * MW_g / AIRMW
+            Spc(N)%Conc(I,J,L) = Ptr3D(I,J,L)
```

Comment:
Can add a `!$OMP COLLAPSE( 3 )` statement to vectorize this loop.

Reply:
Done!